cmd/segment-reaper/delete.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+
+	"storj.io/storj/pkg/process"
+	"storj.io/storj/satellite/metainfo"
+)
+
+var (
+	deleteCmd = &cobra.Command{
+		Use:   "delete",
+		Short: "Deletes zombie segments from DB",
+		Args:  cobra.OnlyValidArgs,
+		RunE:  cmdDelete,
+	}
+
+	deleteCfg struct {
+		DatabaseURL string `help:"the database connection string to use" default:"postgres://"`
+		File        string `help:"location of file with report" default:"detect_result.out"`
```

Comment:
should the default be `delete_result.out`?

Reply:
should be `delete_result.csv`, thanks

Reply:
Yes, I saw it.

I'm not sure if people will be clear that this command reads the file for deleting the segments from the DB.

Even me that I know, I thought in the first review that the `File` was for writing out some result rather than for reading from it.

Reply:
I see your point. I will change default name for `detect` command to `zombie-segments.cvs` and `delete` command will be more like `segment-reaper delete zombie-segmments.csv` (no `File` flag)